src/main/resources/application.yml

```diff
@@ -74,6 +74,10 @@ bankHolidays:
   api:
     url: "https://www.gov.uk"
 
+doc_assembly:
+    url: "http://localhost:4666"
```

Comment:
You may need to add this in infrastructure and charts configurations.

Reply:
👍 added some configs after speaking with @markry-kainos , will ask him to review the changes as well. Thanks @sabahirfan  ;)